tests/model_registry/python_client/test_model_registry_creation_oauth.py

```
@@ -0,0 +1,77 @@
+    """
+    Tests the creation of a model registry with OAuth proxy configuration.
+    Jira ID: RHOAIENG-26194
+    """
+
+    @pytest.mark.smoke
+    def test_registering_model_with_oauth(
+        self: Self,
+        admin_client,
+        model_registry_client: ModelRegistryClient,
+    ):
+        if py_config["distribution"] == "downstream" and MINVER < get_product_version(admin_client=admin_client):
```

Pylint (3.3.7)

[convention] Line too long (113/100)

(C0301)